config/nrfconnect/chip-module/Kconfig

```diff
@@ -202,6 +202,43 @@ config CHIP_FACTORY_RESET_ERASE_NVS
 	bool
 	default y if CHIP_FACTORY_DATA || CHIP_FACTORY_DATA_CUSTOM_BACKEND
 
+config CHIP_WIFI_CONNECTION_RECOVERY_MINIMUM_INTERVAL
```

Comment:
In future we may add some kind of events mechanism to be able to interact with application and decide if e.g. further retries make sense. For now maybe we can at least add sth like `CHIP_WIFI_CONNECTION_RECOVERY_MAX_OVERALL_INTERVAL` which would allow the user to decide if the endless retries are sane for his application or it's better to stop retrying at all after some time.